addons/web/static/src/js/views/kanban/kanban_column_progressbar.js

```diff
@@ -174,18 +174,20 @@ var KanbanColumnProgressBar = Widget.extend({
         // Display and animate the counter number
         var start = this.prevTotalCounterValue;
         var end = this.totalCounterValue;
+        var anim_class = start > 999 ? 'o_kanban_grow' : 'o_kanban_grow_huge';
```

Comment:
- `anim_class` -> `animationClass` (camelCase naming)
- Not sure to understand: why using "huge" when start is small and not "huge" when start is... huge?

Reply:
'huge is related to the animation effect rather than triggering circumstances, so:
number is small > use 'huge' animation to make it visible
number is big    > use 'default' animation hoping that that stuff will not overlap around